packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -2190,43 +2189,24 @@ describe("Garbage Collection Tests", () => {
 			});
 		});
```

Comment:
You can delete the whole describe block "Future GC op type compatibility"

Reply:
Ok I guess keeping the test "process remote op with unrecognized type" somewhere would be good, but it doesn't need to be under a describe block.

Reply:
I'm working on making the test function properly